### PR TITLE
Add the `accessible-placeholder-text` property

### PR DESCRIPTION
**Pull request by DataTriny**

### CODE REVIEW

File: internal/backends/winit/accesskit.rs [hunk 1]
```diff
@@ -437,7 +437,16 @@ impl NodeCollection {
             builder.set_numeric_value_step(step);
         }
 
-        if let Some(value) = item.accessible_string_property(AccessibleStringProperty::Value) {
+        let value = item.accessible_string_property(AccessibleStringProperty::Value);
+        if value.is_none() || value.as_ref().is_some_and(|x| x.is_empty()) {
```
Comment (ogoffart) [inline internal/backends/winit/accesskit.rs:441]: I wonder if this logic shouldn't be better done in the .slint code itself.
Or is there really a reason you can't have a value and a placeholder text at the same time?

**Review by ogoffart — COMMENTED** [review]
Thanks a lot for this nice PR.

Note that the TextEdit also has a placeholder.

My question is whether we should really remove the placeholder if there is a value? Shouldn't the client take care of displaying that?
And if we really should, perhaps it should be the widget responsibility to do `accessible-placeholder-text: text == "" ? "" : placeholder-text;`
What do you think?

File: internal/backends/winit/accesskit.rs (hunk 1 above)
Comment (DataTriny) [reply internal/backends/winit/accesskit.rs]: The tests I performed with both NVDA and Narrator on Windows revealed that the placeholder would still be announced even when the widget had some text. Maybe this is due to Slint not yet implementing editable text, tests will have to be done once you get there.

**Comment by DataTriny** [comment]
I tried passing features `-DSLING_FEATURE_TESTING -DSLING_BUILD_EXAMPLES -DSLINT_FEATURE_EXPERIMENTAL` when compiling with CMAKE yet `ctest -C Debug` only runs 7 tests, not including the ones from the todo example. Unless I am missing something, the CI job logs provide no additional information.

I give up, sorry!

**Comment by tronical** [comment]
> I tried passing features `-DSLING_FEATURE_TESTING -DSLING_BUILD_EXAMPLES -DSLINT_FEATURE_EXPERIMENTAL` when compiling with CMAKE yet `ctest -C Debug` only runs 7 tests, not including the ones from the todo example. Unless I am missing something, the CI job logs provide no additional information.
> 
> I give up, sorry!

No worries, I can try tomorrow and fix it.

**Review by ogoffart — APPROVED** [review]

**Review by tronical — APPROVED** [review]
Thank you :)

**Pull request merged**